pages/_document.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+import { ServerStyleSheet } from 'styled-components/macro'
+
+const ANALYTICS_CODE = `
+  var _paq = window._paq || [];
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="https://arastats.eth.aragon.network/";
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '9']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript';
+    g.async=true;
+    g.defer=true;
+    g.src=u+'matomo.js';
+    s.parentNode.insertBefore(g,s);
+  })();
+`
+
+function AnalyticsScript() {
+  return process.env.NODE_ENV !== 'production' ? null : (
+    <script dangerouslySetInnerHTML={{ __html: ANALYTICS_CODE }} />
+  )
+}
+
+export default class extends Document {
+  static async getInitialProps(ctx) {
+    const sheet = new ServerStyleSheet()
+    const { renderPage } = ctx
+
+    try {
+      ctx.renderPage = () => {
+        return renderPage({
+          enhanceApp: App => props => sheet.collectStyles(<App {...props} />),
+        })
+      }
+
+      const initialProps = await Document.getInitialProps(ctx)
+
+      return {
+        ...initialProps,
+        styles: (
+          <>
+            {initialProps.styles}
+            {sheet.getStyleElement()}
+          </>
+        ),
+      }
+    } finally {
+      sheet.seal()
+    }
+  }
+
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <meta charSet="UTF-8" />
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link rel="icon" href="/favicon.png" />
+          <link
+            rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css"
```

Comment:
Oh interesting, are we still using bootstrap in this site?

Reply:
Yes; the tooltips, overlays and mobile menu are triggered by `react-bootstrap`.